apps/src/code-studio/pd/application/teacher/AdditionalDemographicInformation.jsx

```diff
@@ -57,10 +57,10 @@ const AdditionalDemographicInformation = props => {
                 <p style={styles.error}>
                   We recommend deepening your content knowledge prior to
                   starting this program. This can be accomplished by completing
-                  some additional onboarding prior to attending the CSA
-                  Professional Learning program. Your regional partner will
-                  share this with you after you have been accepted into the
-                  program.
+                  additional pre-work prior to attending the CSA Professional
+                  Learning program. Your regional partner will share this with
+                  you after you have been accepted into the program. This
+                  pre-work usually takes approximately 10 hours.
```

Comment:
Same question for this note: "We recommend deepening your content knowledge prior to starting this program. This can be accomplished by completing additional pre-work that will be shared with you once accepted to the program. This pre-work usually takes approx 10 hours." The content seems pretty close but with a few differences. Just wanted to check if this is the final draft or if the spec has newer changes?

Reply:
The change wasn't highlighted for some reason so I missed it. Great catch!